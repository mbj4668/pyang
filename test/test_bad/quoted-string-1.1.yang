module quoted-string-1.1 {
  yang-version 1.1;
  namespace urn:example:qs;
  prefix qs;

  description
    "Newline is ok.\nAnd tab\t, and a slash\\, and a
     double quote \".";

  container x {
    description
      "But nothing else! \'.";
  }
}

