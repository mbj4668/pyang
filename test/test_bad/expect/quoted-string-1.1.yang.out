quoted-string-1.1.yang:12: error: ILLEGAL_ESCAPE
